py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/converter.py

```diff
@@ -49,59 +51,48 @@ def from_filename(cls, filename: str) -> ConfigType:
             return cls.BUS_VEHICLE_POSITIONS
         if "net_vehicleCount" in filename:
             return cls.VEHICLE_COUNT
+        if "MBTA_GTFS.zip" in filename:
+            return cls.SCHEDULE
 
         raise ConfigTypeFromFilenameException(filename)
 
+    def is_gtfs(self) -> bool:
+        """Is this a GTFS config?"""
+        return self in [self.SCHEDULE]
 
-class ConfigDetail(ABC):
-    """
-    Abstract Base Class for all ConfigDetail implementations.
+    def is_gtfs_rt(self) -> bool:
+        """Is this a GTFS Real Time config?"""
+        return self in [
+            self.RT_ALERTS,
+            self.RT_TRIP_UPDATES,
+            self.RT_VEHICLE_POSITIONS,
+            self.BUS_TRIP_UPDATES,
+            self.BUS_VEHICLE_POSITIONS,
+            self.VEHICLE_COUNT,
+        ]
 
-    ConfigDetail classes must implement all methods and properties that are
-    defined.
-    """
 
-    @property
-    @abstractmethod
-    def config_type(self) -> ConfigType:
-        """Get the config type for this config"""
+class Converter(ABC):
```

Comment:
new `Converter` abstract base class. I don't think anything is too surprising.

if we have a new filetype we want to convert to parquet, we can inherit from this and write a new convert function.